### PR TITLE
Fix Bugzilla 21718 - Preview switches have insufficient descriptions

### PR DESCRIPTION
**Pull request by ntrel**
Add links.
Tweak bitfields description - C-like.
Rewrite vague description of fixImmutableConv.

### CODE REVIEW

File: compiler/src/dmd/cli.d [hunk 1]
```diff
@@ -910,45 +911,64 @@ dmd -cov -unittest myprog.d
...
+            "fix integral promotions for unary + - ~ operators",
+            "https://dlang.org/changelog/2.078.0.html#fix16997", false, true),
         Feature("dtorfields", "dtorFields",
-            "destruct fields of partially constructed objects", false, false),
+            "destruct fields of partially constructed objects",
+            "https://dlang.org/changelog/2.098.0.html#dtorfileds", false, false),
         Feature("rvaluerefparam", "rvalueRefParam",
-            "enable rvalue arguments to ref parameters"),
+            "enable rvalue arguments to ref parameters",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/rejected/DIP1016.md"),
```
Comment (ntrel) [inline compiler/src/dmd/cli.d:953]: I think that is the DIP that Walter is referring to here: https://github.com/dlang/dmd/pull/9817

Given that is marked rejected, perhaps we should add `deprecated: true` for this option.

File: compiler/src/dmd/cli.d [hunk 2]
```diff
@@ -910,45 +911,64 @@ dmd -cov -unittest myprog.d
...
+            "implement Mutable Function Arguments DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md"),
+        Feature("bitfields", "bitfields", "add C-like bitfields",
+            "https://github.com/dlang/dlang.org/pull/3190"),
+        Feature("fieldwise", "fieldwise", "use fieldwise comparisons for struct equality",
+            "https://dlang.org/changelog/2.085.0.html#no-cmpsb"),
         Feature("fixAliasThis", "fixAliasThis",
-            "when a symbol is resolved, check alias this scope before going to upper scopes"),
+            "when a symbol is resolved, check alias this scope before going to upper scopes",
+            "https://github.com/dlang/dmd/pull/8885"),
```
Comment (ntrel) [inline compiler/src/dmd/cli.d:944]: There didn't seem to be a better link to use than this PR. At least it mentions the 2 bugzilla issues it fixes.

File: compiler/src/dmd/cli.d (hunk 1 above)
Comment (dkorpel) [reply compiler/src/dmd/cli.d:953]: It actually implements Andrei's reboot presented at DConf 2019: 
https://gist.github.com/andralex/e5405a5d773f07f73196c05f8339435a

Comment (dkorpel) [reply compiler/src/dmd/cli.d:953]:
```suggestion
            "https://gist.github.com/andralex/e5405a5d773f07f73196c05f8339435a"),
```

**Review by dkorpel — APPROVED** [review]
Just need to update the rvalue dip link

**Review by thewilsonator — APPROVED** [review]

**Comment by thewilsonator** [comment]
... and update `compilable/previewhelp.d` to account for that.

**Pull request merged**